wca-regulations.md

```diff
@@ -493,11 +493,11 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - E2a) The judge distributes a scramble sequence to all competitors. The judge then starts the stopwatch and calls "GO".
     - E2b) All competitors have a total time limit of 60 minutes to devise a solution.
         - E2b1) The judge should call "5 MINUTES REMAINING" at 55 minutes, and must call "STOP" at 60 minutes.
-    - E2c) At 60 minutes, each competitor must give the judge a legibly written solution with the competitor's name, using the notation defined for Outer Block Turn Metric (described in [Regulation 12a](regulations:regulation:12a)). Penalty: disqualification of the attempt (DNF).
-    - E2d) The length of the solution is calculated in Outer Block Turn Metric (see [Regulation 12a](regulations:regulation:12a)).
-        - E2d1) The competitor is permitted a maximum solution length of 80 (moves and rotations).
+    - E2c) At 60 minutes, each competitor must give the judge a legibly written solution with the competitor's name using the notation for NxNxN puzzles (described in [Regulation 12a](regulations:regulation:12a)), such that starting with a solved puzzle and applying the scramble sequence followed by the solution results in a solved puzzle. Penalty: disqualification of the attempt (DNF).
```

Comment:
An unsubmitted or invalid solution is clearly a DNF. Have we historically allowed leniency for missing names if the competitor can be clearly identified after the fact? (I don't recall recent incidents.)

Reply:
I've added a draft solution in https://github.com/thewca/wca-regulations/pull/575/files#diff-8356179f2de288bb7a0aa8a610b9a487R208 that balances practical leniency (you can write your name in front of the judge while submitting your solution) with a clearly enforceable line (you can't add it any later).